src/mds/Anchor.h

```diff
@@ -70,4 +63,10 @@ class OpenedAnchor : public Anchor {
   mutable int nref = 0; // how many children
 };
 
+inline bool operator==(const Anchor &l, const Anchor &r) {
+  return l.ino == r.ino && l.dirino == r.dirino &&
+	 l.d_name == r.d_name && l.d_type == r.d_type;
+}
```

Comment:
Can't this be inside `Anchor` if it should be inline?

Reply:
Yes, it can be. I have made the changes for it.